### DISCUSSION TITLE
Wondering if helm chart docs need updating

### DISCUSSION THREAD

**Discussion by williamclot** [jertel/elastalert2#576] [Ideas]
Hello 👋 ,

I was deploying elastalert2 to k8s using the helm chart and noticed the docs seems to be out of date on specific default values. I was playing around with the TLS settings and thought `elasticsearch.caCerts` defaulted to `/certs/ca.pem` (as per the docs) but in reality it's seems like it has been commented out in the `values.yaml`. I wonder, should the docs be updated to show that they are equal to `NULL` by default?

Thanks a lot,
William

**Comment by jertel** [comment]
Thanks for posting this. I've removed those defaults.
